### PR TITLE
[ZEPPELIN-4884] Support all sql of flink 1.11

### PR DESCRIPTION
**Pull request by lonelyGhostisdog**
### What is this PR for?
Support all sql of flink 1.11

### What type of PR is it?
[ Feature ]

### Todos
* [ ] - Task

### What is the Jira issue?
* [ZEPPELIN-4884] https://issues.apache.org/jira/browse/ZEPPELIN-4884

### How should this be tested?
* travis url https://travis-ci.org/github/lonelyGhostisdog/zeppelin
* Unit Tests & GUI Tests

### Questions:
* No.

### CODE REVIEW

**Comment by alexott** [comment]
Please add link to specific build in Travis-CI, like, https://travis-ci.org/github/lonelyGhostisdog/zeppelin/builds/705348454

Also, one test is failing there..

**Comment by lonelyGhostisdog** [comment]
> Please add link to specific build in Travis-CI, like, https://travis-ci.org/github/lonelyGhostisdog/zeppelin/builds/705348454
> 
> Also, one test is failing there..

Hi, I checked it, the failure not  caused by my commit .

File: flink/interpreter/src/test/java/org/apache/zeppelin/flink/SqlInterpreterTest.java [hunk 1]
```diff
@@ -335,15 +336,15 @@ public void testView() throws InterpreterException, IOException {
     assertEquals(Type.TEXT, resultMessages.get(0).getType());
     assertTrue(resultMessages.get(0).getData(), resultMessages.get(0).getData().contains("already exists"));
 
-    // show tables
+    // show view
```
Comment (zjffdu) [inline flink/interpreter/src/test/java/org/apache/zeppelin/flink/SqlInterpreterTest.java:339]: it is show tables

**Comment by lonelyGhostisdog** [comment]
All tests have passed .travis url https://travis-ci.org/github/lonelyGhostisdog/zeppelin/builds/707090865

**Comment by zjffdu** [comment]
Thanks @lonelyGhostisdog , I am merging

**Pull request closed without merging**